src/EnergyPlus/DaylightingManager.cc

```diff
@@ -5994,7 +5994,7 @@ void manageDaylighting(EnergyPlusData &state)
                     DayltgInterReflIllFrIntWins(state, enclNum);
                     for (int daylightCtrlNum : state.dataDaylightingData->enclDaylight(enclNum).daylightControlIndexes) {
                         auto &thisDaylightControl = state.dataDaylightingData->daylightControl(daylightCtrlNum);
-                        DayltgGlareWithIntWins(state, thisDaylightControl.GlareIndexAtRefPt, enclNum);
+                        DayltgGlareWithIntWins(state, thisDaylightControl.GlareIndexAtRefPt, daylightCtrlNum);
```

Comment:
Looks correct now, not that I know this model but function arguments now agree.

Reply:
I forgot to mention in the description that I ran into this issue randomly while testing the files attached to this defect description.  It was a hard crash because there were six enclosures in the file and only two daylight controls.  When enclNum got higher than the controls number at this point, the wrong index was sent to DayltgGlareWithIntWins and something that was dimensioned to the number of controls tried to reference something beyond what was allocated.  I traced it back to this point and, while I'm not an expert on the daylighting code, this seemed like an obvious error because the subroutine called is clearly using the passed index to access data that was allocated for number of daylight controls, not enclosures.  This change made sense to me.

Reply:
That was my bad, https://github.com/NREL/EnergyPlus/pull/9002, https://github.com/NREL/EnergyPlus/commit/df31c1399335a1f8e2f698f02b2bb29f29248dbf#diff-6e3e1295ba30a23880dcbc9c5ba79c843ccdd4b1a8bd8a646f49044a58b155f9, search for `DayltgGlareWithIntWins`. Incorrect mapping of `NZ` to `daylightCtrlNum` vs `enclNum`.